### ISSUE TITLE
Add auto-completion for sub-commands

### ISSUE THREAD

**Issue by ggardet** [parallaxsecond/parsec-tool#52]
A good enhancement for `parsec-tool` would be the auto-completion for sub-commands.

**Comment by hug-dev** [comment]
Agree! I think there is some support in `clap` to generate autocompletion scripts for some shells like bash. We could produce those scripts and give the instructions how to install them locally.
